packages/canary/canary.0.0.1/opam

```diff
@@ -0,0 +1,48 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "Capture unhandled exceptions and automatically report them through various channels"
+description:
+  "Developed by the University of Zurich, Canary is a tool for capturing unhandled exceptions and automatically reporting them to your development team."
+maintainer: ["John Christopher McAlpine <christophermcalpine@gmail.com>"]
+authors: ["John Christopher McAlpine"]
+license: "MIT"
+homepage: "https://github.com/chrismamo1/canary"
+bug-reports: "https://github.com/chrismamo1/canary/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.08.0"}
+  "lwt"
+  "ptime"
+  "cohttp"
+  "cohttp-lwt"
+  "cohttp-lwt-unix"
+  "yojson"
+  "lwt_ppx"
```

Comment:
```suggestion
  "lwt_ppx" {>= "1.2.2"}
```
Pre `dune` ppxs are often not working nicely with dune. This is the version in which `lwt_ppx` was ported to done